lib/gateways/zendesk_support_tickets.rb

```diff
@@ -0,0 +1,20 @@
+class Gateways::ZendeskSupportTickets
+  def initialize
+    @client = ZendeskAPI::Client.new do |config|
+      config.url = ENV.fetch('ZENDESK_API_ENDPOINT')
+      config.username = ENV.fetch('ZENDESK_API_USER')
+      config.token = ENV.fetch('ZENDESK_API_TOKEN')
+    end
+  end
+
+  def create(subject:, email:, name:, body:)
+    name = 'Unknown' if name.blank?
```

Comment:
We allow this in the form?

Reply:
We do - some of the forms don't even have it.

It's mandatory for the Zendesk API though, if we want to fill in an e-mail, so have to fill it in with something.